indico/web/client/styles/modules/_category.scss

```diff
@@ -445,135 +447,143 @@ ul.category-resource-qtip {
         }
     }
 
-    > table:not(.day) {
-        border: 1px solid #aaa;
-        border-collapse: separate;
-        border-spacing: 1px;
+    .category-table {
+        display: block;
+        overflow-x: auto;
+
+        > table:not(.day) {
+            border: 1px solid #aaa;
+            border-collapse: separate;
+            border-spacing: 1px;
 
-        tr {
-            background-color: #eee;
+            tr {
+                background-color: #eee;
+            }
         }
-    }
 
-    table {
-        width: 100%;
-        border-collapse: collapse;
+        table {
+            width: 100%;
+            border-collapse: collapse;
 
-        th {
-            background-color: #444;
-            color: #fff;
-            text-align: center;
-            font-weight: normal;
-            white-space: nowrap;
-            padding: 1px 15px;
+            th {
+                background-color: #444;
```

Comment:
Is the color not available in `pallate` of colors?

Reply:
There is no `#444`, and also the only thing I changed there was the indentation :) same for all the other ones